src/scss/_main.scss

```diff
@@ -419,18 +419,37 @@ nav {
   &.-left {
     justify-content: flex-start;
   }
+
+  &.-right {
+    justify-content: flex-end;
+  }
 }
 
 .build .error {
   color: var(--color-red-light);
 
   font-size: 16px;
+
+  max-width: 80%;
 }
 
 .build .error .message {
   margin-left: 0.2em;
 }
 
+.log-actions {
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 0.2rem;
```

Comment:
toggling this on/off doesn't change much. can we drop it?

Reply:
Great catch. Done! 👍